### ISSUE TITLE
Actions button text is not visible on hover

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#873]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![20 1](https://user-images.githubusercontent.com/22068434/88356800-e9e3fd80-cd1d-11ea-8692-cdbd68ac61ad.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1382.01. Please regress/close at your convenience.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1383.01, contrast of the text against the background on hover is compliant in all 6 modes
